source/accessibility-statement.html.md.erb

```diff
@@ -0,0 +1,96 @@
+
+We’ve also made the website text as simple as possible to understand.
+
+[AbilityNet](https://mcmw.abilitynet.org.uk/) has advice on making your device easier to use if you have a disability.
+
+### How accessible this website is
+
+We know some parts of this website are not fully accessible.
+
+- some headings do not fit with the content that follows
+- some links do not clearly explain where they lead, or that they lead to external sites
+- there is some inaccessible language
+- some tables are structured in an inaccessible way
+- there are some links to documents in PDF format that are not accessible
+
+### What to do if you cannot access parts of this website
+
+If you need information on this website in a different format like accessible PDF, large print, easy read, audio recording or braille, [select __Report a problem__ on the data.gov.uk Support page](https://data.gov.uk/support) to message the team with details of your request.
+
+We’ll aim to reply within 2 working days.
+
+### Reporting accessibility problems with this website
+
+We’re always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, [select __Report a problem__ on the data.gov.uk Support page](https://data.gov.uk/support) to message the team with details of your request.
+
+### Enforcement procedure
+
+The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’). If you’re not happy with how we respond to your complaint, [the Equality Advisory and Support Service (EASS)][EASS].
+
+## Technical information about this website’s accessibility
+
+GDS is committed to making its website accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
+
+There are parts of this documentation that are not accessible because of [issues caused by our Technical Documentation Template](https://tdt-documentation.london.cloudapps.digital/accessibility).
+
+### Compliance status
+
+This website is partially compliant with the [Web Content Accessibility Guidelines version 2.1](https://www.w3.org/TR/WCAG21/) AA standard, due to the non-compliances listed below.
+
+### Non-accessible content
+
+The content listed below is non-accessible for the following reasons.
+
+#### Non-compliance with the accessibility regulations
+
+* Some headings do not fit with the content that follows, for example [https://guidance.data.gov.uk/publish_and_manage_data/harvest_or_add_data/harvest_data/#dcat-data-json-fields](https://guidance.data.gov.uk/publish_and_manage_data/harvest_or_add_data/harvest_data/#dcat-data-json-fields). This fails WCAG 2.1 success criterion 2.4.6 Headings and labels
```

Comment:
Mixture of list styles - earlier in the document you use `-`, here you use `*`. Could we pick one and keep it consistent?

Reply:
changed to dashes